tutorials/web/resnet.html

```diff
@@ -0,0 +1,103 @@
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <title>NNVM WebGL Test Page</title>
+</head>
+
+<body>
+  <h1>NNVM WebGL Test Page</h1>
+  <img src="data.png">
```

Comment:
Currently pre-processing is done in python side. Can we update the script to use javascript image processing instead? This is likely what most people want to know. reference in https://github.com/dmlc/mxnet.js/blob/master/classify_image.js#L33